services/gateway/apis/http-version3/swagger/definitions/tokens.json

```diff
@@ -39,7 +39,7 @@
                                 "moduleID": {
```

Comment:
We do not have `moduleID` anymore. Is it supposed to be `module` instead? Please double check.

Reply:
applied.